webservices/legal_docs/current_murs.py

```diff
@@ -44,11 +44,13 @@
 """
 
 MUR_DOCUMENTS = """
-    SELECT document_id, filename, category, description, ocrtext,
+    SELECT document_id, mur.case_no, filename, category, description, ocrtext,
```

Comment:
Might be a good opportunity to format the SQL like in AOs, one column per line and preceded by the table name/alias